code/modules/surgery/surgery_step.dm

```diff
@@ -76,7 +77,8 @@
 		implement_speed_mod = implements[implement_type] / 100.0
 
 	speed_mod /= (get_location_modifier(target) * (1 + surgery.speed_modifier) * implement_speed_mod)
-	var/modded_time = time * speed_mod
+	var/modded_time = time * speed_mod * user.mind.get_skill_speed_modifier(/datum/skill/medical)

```

Comment:
put `user.mind.get_skill_speed_modifier(/datum/skill/medical)` into a new var called `good_speed_mods`, then make this line read `var/modded_time = time * speed_mod * good_speed_mods`

put a comment on the good_speed_mods definition that reads "bigger numbers here = faster surgery"

Reply:
adding a var for a single item that isn't changing seems like a waste. It should be obvious that `get_skill_speed_modifier` benefits the time based on your skill.

Also at this point you want smaller times for faster surgeries since you're doing multiplication.

Reply:
ideally in the future further good speed mods could be added into the good_speed_mods definition, even if it is ever so slightly wasteful

Reply:
they can do that when they add multiple benefits :^)